src/Storage/Storage.Management/StorageAccount/NewAzureStorageAccount.cs

```diff
@@ -22,9 +22,15 @@
 using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
 using System;
 using System.Collections.Generic;
+using Microsoft.WindowsAzure.Commands.Common.CustomAttributes;
 
 namespace Microsoft.Azure.Commands.Management.Storage
 {
+    [GenericBreakingChange("Default behavior of AllowBlobPublicAccess will be changed in a future release. The default value of the property will be " +
+        "changed from True to False. With AllowBlobPublicAccess set to False for the Storage account, a container's public access level cannot be set. " +
+        "If you attempt to set the public access level, it will error out indicating that the container's public access level cannot be set.", 
```

Comment:
I think we don't need  "If you attempt to set the public access level, it will error out indicating that the container's public access level cannot be set."

How about change it to:  "Default value of AllowBlobPublicAccess will be changed from True to False in a future release. When AllowBlobPublicAccess is False on a storage account, it is not permitted to configure container ACLs to allow anonymous access to blobs within the storage account. "

Reply:
Yes this one looks good. Thanks!